### DISCUSSION TITLE
YouTube video

### DISCUSSION THREAD

**Discussion by loichustinx** [SafeExamBrowser/seb-win-refactoring#722] [Q&A]
For some reason, certain YouTube videos won't play in the SEB.

The thumbnail is shown, but as soon as you hit the play button it shows an error:
![error](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/115798692/cb23dbe2-3473-4912-84ab-24832de1e285)

Anyone that had this problem before? Or knows where to start looking for a solution?

**Comment by dbuechel** [comment]
SEB for Windows does not support proprietary audio and video codecs, this might be related to that (the reason therefor has been stated numerous times in discussions and issues in this repository).
